src/scss/components/_api-symbol.scss

```diff
@@ -0,0 +1,14 @@
+pulumi-api-symbol {
+    display: flex,
+    align-items: center,
+    justify-content: center,
+    box-sizing: border-box,
+    height: 14px,
+    width: 14,
+    border-width: 1px,
+    border-style: solid,
+    border-radius: 50%,
+    margin-right: 6px,
+    font-weight: 700,
+    font-size: 9,
+}
```

Comment:
I believe some of these can be tailwind-ified - to me, not a merge blocker, but certainly cleaning this up should be added to other theme styling clean up if you don't add it into this.